system_setup/server/controllers/configure.py

```diff
@@ -37,6 +40,90 @@ def __init__(self, app):
     def start(self):
         self.install_task = self.app.aio_loop.create_task(self.configure())
 
+    def _update_locale_gen(self, localeGenPath, lang):
+        """ Uncomments the line in locale.gen file where lang is found,
+            if found commented. A fully qualified language is expected,
+            since that would have passed thru the Locale
+            controller validation. e.g. en_UK.UTF-8. """
+
+        fileContents: str
+        locGenNeedsWrite = False
+        with open(localeGenPath, "r") as f:
+            fileContents = f.read()
+            lineFound = fileContents.find(lang)
+            if lineFound == -1:
+                # An unsupported locale coming from our UI is a bug
+                log.error("Selected language %s not supported.", lang)
+                return
+
+            pattern = r'#+\s*({}.*)'.format(lang)
+            commented = re.compile(pattern)
+            (fileContents, n) = commented.subn(r'\1', fileContents, count=1)
+            locGenNeedsWrite = n == 1
+
+        if locGenNeedsWrite:
+            try:
+                with open(localeGenPath, "wt") as f:
+                    f.write(fileContents)
+            except IOError as err:
+                log.error("Failed to modify %s file. %s", localeGenPath, err)
+
+    async def _install_check_lang_support_packages(self, lang, env):
+        """ Installs any packages recommended by
+            check-language-support command. """
+
+        clsCommand = "check-language-support"
+
+        cp = await arun_command([clsCommand, "-l", lang[0:5]], env=env)
+        if cp.returncode != 0:
+            log.error('Command \"%s\" failed with return code %d',
```

Comment:
hum, do you really need to espace the `"` here? `'` should keep verbatim content

Reply:
Indeed.